conslogging/conslogging.go

```diff
@@ -221,3 +232,19 @@ func (cl ConsoleLogger) color(c *color.Color) *color.Color {
 	}
 	return noColor
 }
+
+func (cl ConsoleLogger) prettyPrefix() string {
+	if cl.prefixPadding == NoPadding {
+		return cl.prefix
+	}
+
+	formatString := fmt.Sprintf("%%%vv", cl.prefixPadding)
```

Comment:
Use the padding number as the number in a format string like this one: "%20v". We just have to format the format string to get there.